public/app/plugins/datasource/postgres/plugin.json

```diff
@@ -18,5 +18,10 @@
 
   "alerting": true,
   "annotations": true,
-  "metrics": true
+  "metrics": true,
+
+  "queryOptions": {
```

Comment:
Don't think this is needed right now. The $__interval should work anyway. Think it's for calculating intervals on backend.